src/test/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/services/OcrFieldRetrieverTest.java

```diff
@@ -0,0 +1,159 @@
+package uk.gov.hmcts.reform.bulkscanccdeventhandler.services;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.model.CcdCollectionElement;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.model.OcrData;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.model.OcrDataField;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.model.ResultOrErrors;
+
+import java.time.LocalDate;
+import java.util.Arrays;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.toList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertFalse;
```

Comment:
no need for those if you're using assertJ already

Reply:
Oh great. What a mix. Will fix.

Reply:
Fixed